pkg/build/builder/docker_test.go

```diff
@@ -338,6 +344,13 @@ RUN echo "hello world"
 			want: []string{"scratch", "busybox"},
 		},
 	}
+
+	if isNew, err := util.CreateBaseDir(util.GetBaseDir()); err == nil && isNew {
+		defer os.RemoveAll(util.GetBaseDir())
+	} else {
+		t.Errorf("failed to create BaseDir: %v", err)
+	}
```

Comment:
this needs to be reworked like the other one so you don't error out if isNew==false.